logging/logging.go

```diff
@@ -628,6 +630,9 @@ func jsonValueToStructValue(v interface{}) *structpb.Value {
 // and will block, it is intended primarily for debugging or critical errors.
 // Prefer Log for most uses.
 func (l *Logger) LogSync(ctx context.Context, e Entry) error {
+	if e.Severity < l.minimalSeverity {
```

Comment:
nit: why not `<=`? Inclusive threshold is implied by the name: "minimal severity" -> "severity below which nothing is ingested"
nit: should it return a specialized error indicating that the ingestion did not take place? returning `nil` means everything is OK but the job still was not done.

Reply:
> nit: why not <=? Inclusive threshold is implied by the name: "minimal severity" -> "severity below which nothing is ingested"

I came from the definition of a minimum: https://en.wikipedia.org/wiki/Maxima_and_minima. The minimal severity is the minimum allowed severity value. That's why I used `<` in the expression.

> nit: should it return a specialized error indicating that the ingestion did not take place? returning nil means everything is OK but the job still was not done.

An error is defined as a difference between the desired and actual behavior. The minimal severity has desired behavior, that's why nil is returned.